anaconda.py

```diff
@@ -303,7 +303,9 @@ def setup_environment():
 
     # warn users that they should use inst. prefix all the time
     for arg in depr:
-        stdout_log.warning("Boot argument '%s' is deprecated and will be removed in the future. "
+        stdout_log.warning("Boot argument '%s' has been deprecated same as all Anaconda boot "
+                           "arguments without 'inst.' prefix. "
+                           "They will be removed with a future major release. "
```

Comment:
How about:

_Boot argument '%s' must be used with the 'inst.' prefix. Please use 'inst.%s' instead. All usage of Anaconda boot arguments without 'inst.' prefix has been deprecated and will be removed in a future major release._

That said, I wonder if you could leave out the sentence "All usage..." and print it only once at the end, if `depr` is not empty. It would drastically improve readability of the output, as the warnings would probably fit on a single line.